include/graph.h

```diff
@@ -5,13 +5,17 @@
 #include <set>
 #include <fstream>
 #include "supernode.h"
+#include <atomic>  // REMOVE LATER

```

Comment:
hm

Reply:
Yeah so this is for that counter that I have... if you think it should go I'm happy to remove it.